### PR TITLE
DOCS-1561: Make installation guide and component page for Orange Pi Zero2 and `orangepi`

### PR DESCRIPTION
**Pull request by sguequierre**
* Make Orange Pi Zero2 setup/installation guide
* Make component page for orangepi
* Add Orange Pi Zero2 to boards carousel

### CODE REVIEW

File: docs/get-started/installation/prepare/orange-pi-zero2.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+To connect to a display, connect the micro-HDMI end of your HDMI cable to the Orange Pi and the other end to your monitor.
+Then, to connect the keyboard and mouse, plug the two devices into your USB hub and connect the USB hub to your Orange Pi's USB-A port.
+
+You should now be greeted with the Orange Pi display.
+If it prompts you for a password, note that the default password for Orange Pi devices is "orangepi".
+
+## Establish a network connection
+
+The Orange Pi Zero2 comes equipped with a wireless network antennae.
```
Comment (npentrel) [inline docs/get-started/installation/prepare/orange-pi-zero2.md:59]:
```suggestion
The Orange Pi Zero2 comes equipped with a wireless network antenna.
```
I think?

**Review by npentrel — APPROVED** [review]
Thanks for getting these done Sierra!

File: layouts/shortcodes/board-carousel.html [hunk 2]
```diff
@@ -38,60 +38,66 @@
       </a>
     </li>
     <li id="c1_slide6">
+      <a href="/get-started/installation/prepare/orange-pi-zero2/">
```
Comment (npentrel) [inline layouts/shortcodes/board-carousel.html:41]: Do we need the other orange pi here too?

**Review by npentrel — COMMENTED** [review]
Sorry one more comment

File: layouts/shortcodes/board-carousel.html (hunk 2 above)
Comment (npentrel) [reply layouts/shortcodes/board-carousel.html:41]: Oh i thought this was also covering the orange pi 3 ignore me

**Review by oliviamiller — APPROVED** [review]

**Review by oliviamiller — APPROVED** [review]

**Pull request merged**